enterprise/coderd/coderd.go

```diff
@@ -77,10 +77,18 @@ func New(ctx context.Context, options *Options) (*API, error) {
 		r.Route("/organizations/{organization}/groups", func(r chi.Router) {
 			r.Use(
 				apiKeyMiddleware,
+				api.templateRBACEnabledMW,
```

Comment:
Should `GET` still work? Just making new groups and editing groups disabled?

Reply:
We can do that in the future if it becomes an issue. I'm just maintaining the status quo.